doc/stdenv/passthru.chapter.md

```diff
@@ -0,0 +1,253 @@
+# Passthru-attributes {#chap-passthru}
+[]{#var-stdenv-passthru} []{#special-variables} <!-- legacy anchors -->
+
+`passthru` is an attribute set which can be filled with arbitrary values. 
+For example:
+
+```nix
+{
+  passthru = {
+    foo = "bar";
+    baz = {
+      value1 = 4;
+      value2 = 5;
+    };
+  };
+}
+```
+
+Much of the passthru's power comes from two features:
+- You can access passthru attributes as if they were set inside a derivation (e.g. `hello.baz.value1`).
+- These values are not passed to the builder, so you can change them without triggering a rebuild.
+
+One example of the passthru's power is how it enables us to associate the package for an interpreter Python or Ruby with a set of modules built for that specific interpreter (e.g., `python.pkgs` and `ruby.gems`) and to build runtime environments with a collection of those modules (e.g., (`python.withPackages`)[#python.withpackages-function] and `ruby.withPackages`).
+<!-- python.pkgs, ruby.gems, and ruby.withPackages are mentioned but don't have dedicated sections to link -->
```

Comment:
Comment here is just a reminder for why I only linked one of these :)

Reply:
Is this trying to say that `python.pkgs` and `python.withPackages` are defined in `python.passthru`? If so it should be explicit.

Reply:
@kjeremy Can you clarify how that could be more explicit? (Perhaps in the form of a rephrasing? I feel like the text is already explicitly stating that the passthru is where these are defined.)